projects/kit/src/components/counter/components/counter/counter.component.html

```diff
@@ -3,6 +3,14 @@
     {{ count }}
   </ng-container>
   <ng-container *ngSwitchDefault>
-    {{ count | formatCount }}
+    <ng-container [ngTemplateOutlet]="maxCount ? moreThanCountFormat : abbreviatedCountFormat"></ng-container>
```

Comment:
внутри formatCountPipe есть проверка на isNil второго аргумента. Так что можно просто оставить вот так
```suggestion
    {{ count | formatCount : maxCount }}
```

Reply:
Да, спасибо, поправил